bundles/action/org.openhab.action.xmpp/README.md

```diff
@@ -43,3 +43,14 @@ password=mysecret
 # check you openhab.log to found the address after you send something via Hangouts to your openHAB account
 consoleusers=**cryptic**@public.talk.google.com,myname@gmail.com
 ```
+
+## Generate PIN

```

Comment:
Please update to say `Example: Generate PIN`.

Reply:
done